external/src/main/java/com/redhat/cloud/policies/engine/RunOnStartup.java

```diff
@@ -0,0 +1,56 @@
+package com.redhat.cloud.policies.engine;
+
+import io.quarkus.runtime.Startup;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.regex.Pattern;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.annotation.PostConstruct;
+import org.jboss.logging.Logger;
+
+@Startup
+public class RunOnStartup {
+
+    // TODO POL-649 Update that pattern after the new health check call has been confirmed.
```

Comment:
I'm supposing this is to ease the updating to the new API ?

Reply:
Yes, this can be done later.

Before the lightweight engine, `ui-backend` called the `engine` getTrigger REST endpoint as part of its health check. I replaced that with the only remaining `engine` REST endpoint with is `/validate`. But I don't really like that. I think we should have a dedicated `engine` endpoint for the `ui-backend` health check. It could be something as simple as `GET /ping`. The pattern here will need an update after we're done chosing the best REST call.

Reply:
Filtering all `/validate` calls wouldn't make sense as some of them will be real requests that don't come from the health check.